RestaurantMap/Extentions/UIImageView+Extentions.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  UIImage+Extention.swift
+//  RestaurantMap
+//
+//  Created by Viet Phan on 15/04/2022.
+//
+
+import Foundation
+import Kingfisher
+import Photos
+
+let imageCache = NSCache<NSString, UIImage>()
+
+extension UIImageView {
+    
+    func setImage(url: URL?, placeholder: UIImage?, resize: CGSize = .zero, isShowLoading: Bool = true, isForceReload: Bool = false, completion: (() -> Void)? = nil) {
+        if isShowLoading {
+            self.kf.indicatorType = .activity
+        }
+        if let url = url {
```

Comment:
Should remove this if else command, because the purpose you want to do isn't depended on the result of if or else.   We shouldn't compare that if has url we will create and pass a new option when set image, or not url is nill will set image with a nil url and no options. That's wrong logic.
Solution:
 Remove if  else and use  only piece of code inside the if .